oozie/src/main/java/org/apache/falcon/workflow/engine/OozieClientFactory.java

```diff
@@ -55,7 +55,7 @@ public static OozieClient get(String clusterName) throws FalconException {
         return get((Cluster) ConfigurationStore.get().get(EntityType.CLUSTER, clusterName));
     }
 
-    private static OozieClient getClientRef(String oozieUrl)
+    public static OozieClient getClientRef(String oozieUrl)
```

Comment:
Instead of exposing this method, JobLogMover should use OozieClientFactory.get(context.getClusterName).